README.md

```diff
@@ -44,13 +46,25 @@ The following instructions deploy Mattermost in a production configuration using
     environment:
       - MATTERMOST_ENABLE_SSL=false
     ```
-    
+
 2. Build and run mattermost
 
+    ```
     docker-compose up -d
+    ```
 
 3. Open `http://your.domain` with your web browser.
 
+### Install the Enterprise Version
```

Comment:
Propose:

1 - we install the Enterprise version by default and give users the option to install team edition only.

This is how the server binaries are given on https://about.mattermost.com/download, so it would be good to be consistent.

2 - move these instructions above `Install with SSL certificate` and simply have a note along the lines of:

This Docker compose installs the [Enterprise Edition server](https://docs.mattermost.com/administration/upgrade.html#mattermost-enterprise-edition-server) by default. If you'd like to install a [Team Edition server](https://docs.mattermost.com/administration/upgrade.html#mattermost-team-edition-server), comment out the following line in `docker-compose.yaml` file.

```
dockerfile: Dockerfile-enterprise
```

```
This Docker compose installs the [Enterprise Edition server](https://docs.mattermost.com/administration/upgrade.html#mattermost-enterprise-edition-server) by default. If you'd like to install a [Team Edition server](https://docs.mattermost.com/administration/upgrade.html#mattermost-team-edition-server), comment out the following line in `docker-compose.yaml` file.

dockerfile: Dockerfile-enterprise
```

Wondering if this should come earlier in the install instructions